This gets confused by python-any-r1 and tries to find sphinx-build in ${T}.
--- a/docs/meson.build
+++ b/docs/meson.build
@@ -1,5 +1,4 @@
-sphinx_build = find_program(fs.parent(python.full_path()) / 'sphinx-build',
-                            required: get_option('docs'))
+sphinx_build = find_program('sphinx-build', required: get_option('docs'))
 
 # Check if tools are available to build documentation.
 build_docs = false
